duckduckgo/macos-browser
# Implements solution for System Extension activation issues

Task/Issue URL: https://app.asana.com/0/0/1205275221447702/f

## Description:

This PR offers a solution to the System Extension activation issues we currently have.
It also modifies the startup flow: now when the user enables the system extension they'll still need to start the tunnel manually.

## Steps to test this PR:

1. Launch the app and reset the Network Protection Status
2. Start the onboarding, see that the system-extension-blocked alert is shown.
3. Ignore the request, tap on the button to start the onboarding again.
4. Make sure system settings is opened.

---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
